rtt_roscomm/include/rtt_roscomm/passthrough_callback_queue.hpp

```diff
@@ -0,0 +1,232 @@
+
+#ifndef RTT_ROSCOMM__PASSTHROUGH_CALLBACK_QUEUE_HPP_
+#define RTT_ROSCOMM__PASSTHROUGH_CALLBACK_QUEUE_HPP_
+
+#include <ros/ros.h>
+#include <ros/callback_queue_interface.h>
+#include <ros/subscription_queue.h>
+#include <ros/callback_queue.h>
+
+namespace rtt_roscomm {
+  class PassthroughCallbackQueue: public ros::CallbackQueueInterface
+  {
+    public:
+      PassthroughCallbackQueue();
```

Comment:
It is trivial, so not needed to define a constructor explicitly at all.
If you do, you should probably also declare an explicit `virtual` destructor (the implicitly generated one is `virtual`, too).